zulipterminal/config/themes.py

```diff
@@ -49,6 +48,10 @@
     'area:msg': 'standout',
     'area:stream': 'standout',
     'area:error': 'standout',
+    'footer:neutral': 'standout',
+    'task:success': 'standout',
+    'task:failure': 'standout',
+    'task:warning': 'standout',
```

Comment:
I think having a color (say green) on a result (say success) draws user's attention more than just changing the text with the default footer background. We could surely gain feedback in stream in any case.

Reply:
Right, my initial thought would be to temporarily do something like invert from the default style; I'm not sure what that translates to. Discussing on the stream would be good, though this would be a small update later too.